PatientRegistrationApp/src/components/HouseholdPersonalInfo_n.vue

```diff
@@ -115,8 +115,8 @@
       <v-col class="d-flex" cols="5" sm="5">
         <!-- Current Photo -->
         <v-file-input
-          :rules="rules"
           accept="image/png, image/jpeg, image/bmp"
+          :rules="[(v) => !v || v.size < 2097152 || 'Image size should be less than 2 MB!']"
```

Comment:
The rules got messed up and I can't continue without uploading a photo (which shouldn't be required). My workflow was:

1. Run household for 3 people
2. Fill out info but don't add a photo
3. On personal info page (where you can upload a photo but I didn't) I can't move forward but am not given an alert as to what's missing

Reply:
The rules also seem off in the household workflow since I can proceed forward with a photo >2MB

Reply:
Tested to see if you can still proceed in household workflow when putting an image over 2mb's, it no longer works.